### PR TITLE
More css touch up

### PR DESCRIPTION
**Pull request by Tristan-Holtz**
Got the dropdown menu looking right and the cards are sitting where they should. Also added border radius where needed and fixed the search icon scaling.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -113,33 +114,39 @@ textarea {
 
 input {
   padding: 10px;
+  border-radius: 4px;
```
Comment (HartiganHM) [inline styles.css:116]: Make sure you're holding onto your convention here. If you're ordering your CSS properties alphabetically, this should go above padding.

File: styles.css [hunk 2]
```diff
@@ -113,33 +114,39 @@ textarea {
...
 }
 
 textarea {
   height: 100px;
   overflow: scroll;
+  border-radius: 4px;
```
Comment (HartiganHM) [inline styles.css]: Same here with sorting alphabetically.

File: styles.css [hunk 3]
```diff
@@ -113,33 +114,39 @@ textarea {
...
 }
 
 .save {
   color: #BFBFDD;
-  padding: 16px;
+  padding: 12px;
   font-size: 24px;
   font-family: 'Open sans';
   font-weight: bold;
+  border-radius: 4px;
```
Comment (HartiganHM) [inline styles.css]: Same here with sorting alphabetically.

File: styles.css [hunk 4]
```diff
@@ -113,33 +114,39 @@ textarea {
...
 }
 
 .disable-save-btn {
   background-color: #585759;
 }
 
-.active-save-btn {
+.save.active-save-btn {
```
Comment (HartiganHM) [inline styles.css:139]: Nice job combining these two classes for greater specificity! 🔥

File: styles.css [hunk 5]
```diff
@@ -256,6 +269,7 @@ textarea {
 
   body {
     flex-flow: column nowrap;
+    z-index: 1;
```
Comment (HartiganHM) [inline styles.css]: Why are you adding a `z-index` to the entire `body` element? This seems a little risky. Is there a way to execute what you're trying to do without it?

**Review by HartiganHM — COMMENTED** [review]
Overall it looks like you tidied up a lot of styles, great job! I'm not sure if you've been ordering your CSS properties in a particular way or not, but if it's alphabetically, make sure you're keeping that consistent with each PR.

File: styles.css (hunk 2 above)
Comment (Tristan-Holtz) [reply styles.css]: On it, thanks for the spot, I'd rather fix it now

File: styles.css (hunk 5 above)
Comment (Tristan-Holtz) [reply styles.css]: I was making sure the dropdown menu has a higher index than the body its covering, I can try to take it off and see if everything is still working right

File: styles.css (hunk 4 above)
Comment (Tristan-Holtz) [reply styles.css:139]: That was harder than it should've been, haha.

File: styles.css (hunk 5 above)
Comment (Tristan-Holtz) [reply styles.css]: Everything looks good still, thanks for the tip

**Review by HartiganHM — APPROVED** [review]
Looks great! 👍

**Pull request merged**